src/database/schema/ChatSchema.ts

```diff
@@ -224,7 +225,7 @@ class ChatSchema implements BaseDbSchema {
       channelId,
       userId: json?.user?.id,
       message: (json?.text || json?.message) ?? '',
-      type: json?.type,
+      type: json?.message_type ?? json?.type,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the comment on line 202, this change is to handle the case where `message_type` might not be present in `json`. It's important to ensure that the fallback value (`json?.type`) is always available or else it could lead to unexpected behavior.
```diff
-      type: json?.type,
+      type: json?.message_type ?? json?.type,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->